src/Installers/Windows/SharedFramework/Product.wxs

```diff
@@ -109,6 +115,12 @@
                 </RegistryKey>
             </Component>
             <?endif?>
+
+            <Component Id="C_ProductCentralVersion">
+                <RegistryKey Key="$(var.ProductVersionCentralKey)" Root="HKLM">
+                    <RegistryValue Name="Version" Type="string" Value="$(var.Version)" />
```

Comment:
The registry key should include `Microsoft.AspNetCore.App`. the registry value needs to be the version and the data should be a type DWROD set to 1.